### ISSUE TITLE
Can't bind to 'template' since it isn't a known property of 'ng-template'.

### ISSUE THREAD

**Issue by kiebor** [apoterenko/ngx-dynamic-template#56]
Can't bind to 'template' since it isn't a known property of 'ng-template'.

I have my module set up as follows:

```
@NgModule({
  imports: [
    CommonModule,
    HttpModule,
    ComponentModule,
    NgxDynamicTemplateModule.forRoot(
      {
        extraModules: [
          RoutingModule,
          ComponentModule
        ],
        routes: routes,
        useJit: true
      }
    ),
  ]
})
export default class AreaModule { }
```

the `ComponentModule` is a simple module that imports and exports all the Components I want to use for this area of my application. They have default templates which are precompiled with the usual bundles (angular cli). 

The exception is thrown if I embed any template with `<ng-template>` as follows where `template` and `context` are defined in the component.
```
<ng-template
             dynamic-template
             [template]="template"
             [context]="context">
</ng-template>
```
Basically the `dynamic-template` directive is not recognized. However, if I add `NgxDynamicTemplateModule` to my `ComponentModule`:

```
    NgxDynamicTemplateModule.forRoot(
      {
        extraModules: [
          RoutingModule
        ],
        routes: routes,
        useJit: true
      }
    )
```
The exception no longer gets thrown at runtime. Is this by design that I have to recursively add `NgxDynamicTemplateModule` to the imports array of every module I want to use?

**Comment by apoterenko** [comment]
Hi! 

<< The exception no longer gets thrown at runtime. Is this by design that I have to recursively add 

No, this is a bug. See the updated demo at 2.2.3
